### ISSUE TITLE
SDXL weird blue and red points on image

### ISSUE THREAD

**Issue by ousinkou** [huggingface/diffusers#4074]
![image](https://github.com/huggingface/diffusers/assets/13220463/d775a599-e389-4c78-b2b4-6a7f73a1fd4b)
I use diffusers to inference sdxl 0.9 checkpoints, the output image is ok, but I found blue and red points on image, it's weird.
I have used fp16, enable_model_cpu_offload, enable_vae_slicing, enable_vae_tiling. Anyone help?

**Comment by ousinkou** [comment]
#4035

**Comment by patrickvonplaten** [comment]
Yes it's the watermarker - we should try to get https://github.com/YuxinWenRick/tree-ring-watermark working
